app/konnect/centralized-consumers.md

```diff
@@ -0,0 +1,129 @@
+direction LR
+    A
+    B
+end
+subgraph id2 [Dev realm]
+direction LR
+    G
+end
+
+
+id1 --> id3
+id1 --> id4
+id2 --> id5
+
+subgraph id3 [Appointments Control Plane]
+direction LR
+C
+D
+end
+
+subgraph id4 [Billing Control Plane]
+direction LR
+H
+I
+end
+
+subgraph id5 [Dev Control Plane]
+direction LR
+F
+E
+end
+
+style id1 stroke-dasharray:3,rx:10,ry:10
+style id2 stroke-dasharray:3,rx:10,ry:10
+style id3 stroke-dasharray:3,rx:10,ry:10
+style id4 stroke-dasharray:3,rx:10,ry:10
+style id5 stroke-dasharray:3,rx:10,ry:10
+{% endmermaid %}
+<!-- vale on-->
+
+## Create centralized Consumers and Realms 
+
+You can create centralized Consumers using the {{site.konnect_short_name}} API. Only Org Admins have CRUD permissions for centralized Consumers. 
+
+1. Use the following custom {{site.base_gateway}} image when you create a [Data Plane node](/konnect/gateway-manager/data-plane-nodes/):
+   ```
+   kong/kong-gateway-dev
+   ```
+1. Use the `/realms` endpoint to create a realm and optionally associate it with allowed Control Planes and time-to-live values:
+   ```
+   curl -X POST \
+   https://{region}.api.konghq.com/v1/realms \
+   -H "Content-Type: application/json" \
+   -H "Authorization: Bearer $KONNECT_TOKEN" \
+   -d '{
+        "name": "prod",
+        "allowed_control_planes": [
+            "$CONTROL_PLANE_UUID"
+        ],
+        "ttl": 10,
+        "negative_ttl": 10,
+        "consumer_groups": [
+            "$CONSUMER_GROUP"
+        ]
+    }'
+   ```
+   Save the ID of the realm.
+
+   Be sure to replace the following with your own values:
+   * {region}: Region for your {{site.konnect_short_name}} instance.
+   * $KONNECT_TOKEN: Replace with your {{site.konnect_short_name}} personal access token.
+   * $CONTROL_PLANE_UUID: (Optional) Replace with your Control Plane UUID.
+   * `ttl` and `negative_ttl`: (Optional) Time-to-live in minutes of the consumer or negative consumer for this realm in the {{site.base_gateway}} cache.
+   * $CONSUMER_GROUP: (Optional) Replace with the name of the consumer groups you want to associate with the realm.
+1. Use the `/realms/{realmId}/consumers` endpoint to create a Consumer and optionally assign it to a Consumer Group:
+   ```
+   curl -X POST \
+   https://{region}.api.konghq.com/v1/realms/{realmId}/consumers \
+   -H "Content-Type: application/json" \
+   -H "Authorization: Bearer TOKEN" \
+   -d '{
+         "username": "$CONSUMER_NAME",
+         "consumer_groups": ["$CONSUMER_GROUP"]
+       }'
+   ```
+   Be sure to replace the following with your own values:
+   * {region}: Region for your {{site.konnect_short_name}} instance.
+   * $KONNECT_TOKEN: Replace with your {{site.konnect_short_name}} personal access token.
+   * {realmId}: The ID of the realm you created previously. 
+   * $CONSUMER_NAME: Replace with the name of the consumer.
+   * $CONSUMER_GROUP: (Optional) Replace with the name of the consumer groups you want to associate with the consumer.
```

Comment:
If I specify `consumer_groups` on a consumer does that replace `consumer_groups` from the realm, or are they added together?

If I specify `consumer_groups: []` does that mean no consumer groups, or use the groups from the realm?

Reply:
This is additive here, I'll add that info to the doc.